data_changes.md

```diff
@@ -0,0 +1,58 @@
+## Changes to Imported Data ##
+
+Generally, data will be imported as-is from historical data or newly given data
+with a few exceptions:
+
+- _IMPORT TYPE_: Broker, USASpending
+- _CHANGE_: **Creation of summary awards**
+- _DESCRIPTION_: Parent awards that do not currently exist in the database will be
+created with empty data, to provide references for transactions being loaded.
+- _FILE_: `submissions/management/commands/load_submission.py`, et. al.
+
+
+- _IMPORT TYPE_: Broker
+- _CHANGE_: **State code converted to foreign province**
+- _DESCRIPTION_: Cases where a location must be created, but the data specified has
```

Comment:
I mean, what's the target audience here? Are we assuming people reading about changes to incoming data during an ETL are having no knowledge of databases at all?